extensions/amp-accordion/1.0/accordion.js

```diff
@@ -316,10 +321,13 @@ export function AccordionSection({
       /** @type {AccordionDef.SectionContextProps} */ ({
         animate,
         contentId,
+        headerId,
         expanded,
         expandHandler,
+        registerContentId: setContentIdState,
```

Comment:
Nit: "register" sounds to me like there are many. "set" is more for one. So I think it'd be very reasonable to call this `setContentId` and `setHeaderId`.

Reply:
Sounds good, updated in the latest revision.